public_html/wp-content/plugins/pattern-directory/includes/logging.php

```diff
@@ -9,8 +9,39 @@
 /**
  * Actions and filters.
  */
+add_filter( 'wporg_internal_notes_rest_prepare_response', __NAMESPACE__ . '\replace_rest_controller_author_link' );
```

Comment:
Why not make this change in the internal notes plugin itself?

Reply:
Good question. I originally thought of putting it there, but decided it would be best to keep the functionality in the plugin as environment-neutral as possible (which was inspired by @dd32 mentioning that he had tried using the plugin elsewhere on an external site). That said, all uses of the Internal Notes plugin I can think of within wporg would probably need this same modification, so perhaps this would be better somewhere in mu-plugins?

Reply:
Oh, I misread the endpoint namespace change in https://github.com/WordPress/wporg-internal-notes/pull/12, and thought it was already using wporg endpoints. Since it's not designed for wporg-only, I take back my comment :)

> all uses of the Internal Notes plugin I can think of within wporg would probably need this same modification, so perhaps this would be better somewhere in mu-plugins?

Yeah, that makes sense to me.